### PR TITLE
Update kubernetes container add severity

### PR DESCRIPTION
**Pull request by ericwholt**
When the kubernetes plugin was split the severity parser was not added to kubernetes_container. Added severity parser to parse from stream field.
- Add severity parser
- Move log field to message field

### CODE REVIEW

**Comment by jsirianni** [comment]
Still getting default severity with this plugin in my testing (minikube) environment. Will comment back when it is working.

**Review by jsirianni — APPROVED** [review]
Looks good in my minikube, and confirmed it is working with Eric's minikube.

File: plugins/kubernetes_container.yaml [hunk 1]
```diff
@@ -101,5 +107,5 @@ pipeline:
           to: "$labels.stream"
       - move:
           from: log
-          to: "$record"
+          to: "$record.message"
```
Comment (andykellr) [inline plugins/kubernetes_container.yaml:110]: Does this make a difference? I think $record is fine.

**Review by andykellr — APPROVED** [review]

File: plugins/kubernetes_container.yaml (hunk 1 above)
Comment (ericwholt) [reply plugins/kubernetes_container.yaml:110]: It shouldn't, but the ticket specified message.

**Pull request merged**